gdal/ogr/ogrsf_frmts/flatgeobuf/ogrflatgeobuflayer.cpp

```diff
@@ -142,12 +142,13 @@ OGRFlatGeobufLayer::OGRFlatGeobufLayer(
     VSILFILE *poFpWrite,
     std::string oTempFile,
     bool bCreateSpatialIndexAtClose) :
-    m_create(true),
     m_eGType(eGType),
     m_poFpWrite(poFpWrite),
-    m_oTempFile(oTempFile),
-    m_bCreateSpatialIndexAtClose(bCreateSpatialIndexAtClose)
+    m_oTempFile(oTempFile)
 {
+    m_create = true;
```

Comment:
that's fine, but reordering the order of your initialization to be the same as of their declaration would have also fixed it

Reply:
I tried and failed at that. :rofl: